TranslatorReasonerAPI.yaml

```diff
@@ -627,7 +628,9 @@ components:
             The list of all Analysis components that contribute to the result.
             See below for Analysis components.
           items:
-            $ref: '#/components/schemas/Analysis'
+            oneOf:
+              - $ref: '#/components/schemas/Analysis'
+              - $ref: '#/components/schemas/PathfinderAnalysis'
```

Comment:
Is there any way we can tie together these oneOf choices so that a developer is forced to pick between the different versions? It seems like the way its specified you could mix using Analysis with PathfinderQueryGraph for instance.

Reply:
I guess it would have to be done higher up the chain. So we could have a Message and PathfinderMessage, such that PathfinderMessages would only have PathfinderQueryGraphs and PathfinderAnalysis. Such that one XMessage is committed to one or the other.

Reply:
Consensus from 2025-05-15 call: consensus is that it would be best to leave the schema as proposed because of future intended changes. Handle the risk of improper use with the semantic validator which can check that it is not used in a unintended way (e.g. combining Analysis and PathfinderAnalysis).